app/components/subject-viewer.cjsx

```diff
@@ -260,6 +267,9 @@ module.exports = createReactClass
       mods[type] = not mods[type]
     @setState modification: mods
 
+  toggleInvert: (invert) ->
```

Comment:
Not a blocking comment, but this could probably be merged into the `toggleModification` handler.